pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -156,13 +160,9 @@ func (p *Plugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (*spi
 		return nil, newError("trust_domain is required")
 	}
 
-	if config.CABundlePath == "" {
-		return nil, newError("ca_bundle_path is required")
-	}
-
-	trustBundle, err := util.LoadCertPool(config.CABundlePath)
+	bundles, err := getBundles(p.log, config)
```

Comment:
Hmm, i wonder if there are any practical implications for lumping all of these into the same cert pool for validation....

Reply:
I was wonder about the same thing, but is not it the same when some one attach system certs for validations?

At the same time, if we run a validation with each file, it will result in multiple error outputs, that we may or not log, I was worry about not providing enough information on an error returned after that validation or spam in our logs in case we want to log all